File: Fibonacci/Fibonacci.go
package main

import (
	"fmt"
)

func fibonacci(n int) []int {
	if n <= 2 {
		return []int{} // Devuelve una lista vacía si n <= 2
	}

	sequence := []int{1, 1}
	for i := 2; i < n; i++ {
		sequence = append(sequence, sequence[i-1]+sequence[i-2])
	}

	return sequence
}

func main() {
	var n int
	fmt.Print("Ingrese un número mayor que 2: ")
	fmt.Scan(&n)

	if n > 2 {
		fmt.Println("Secuencia de Fibonacci:", fibonacci(n))
	} else {
		fmt.Println("Error: El número debe ser mayor que 2.")
	}
}

File: Fibonacci/fibonacci.cs
using System;
using System.Collections.Generic;

class Program
{
    static void Main()
    {
        Console.Write("Ingrese un número mayor que 2: ");
        if (int.TryParse(Console.ReadLine(), out int n) && n > 2)
        {
            List<int> fibonacciSequence = Fibonacci(n);
            Console.WriteLine("Secuencia de Fibonacci:");
            Console.WriteLine(string.Join(", ", fibonacciSequence));
        }
        else
        {
            Console.WriteLine("Entrada inválida. Debe ingresar un número mayor que 2.");
        }
    }

    static List<int> Fibonacci(int n)
    {
        if (n <= 2) return new List<int>();

        List<int> sequence = new List<int> { 1, 1 };
        for (int i = 2; i < n; i++)
        {
            sequence.Add(sequence[i - 1] + sequence[i - 2]);
        }
        return sequence;
    }
}
